src/components/block/index.ts

```diff
@@ -541,7 +541,7 @@ export default class Block extends EventsDispatcher<BlockEvents> {
    * @param {BlockToolData} data - data to merge
    */
   public async mergeWith(data: BlockToolData): Promise<void> {
-    await this.toolInstance.merge(data);
+    this.toolInstance.merge(data);
```

Comment:
Is there a reason why `await` has been removed?

Reply:
Must be my IDE suggesting thing, but we should put the type to `Promise<void> | void` 🤔